easy_phi/app.py

```diff
@@ -210,19 +236,16 @@ def get(self):
         self.write("Coming soon..")
 
 
-def main(application):
-    if __name__ == '__main__':
-        # parse command line twice to allow pass configuration file path
-        parse_command_line(final=False)
-        parse_config_file(options.conf_path, final=False)
-        # command-line args override conf file options
-        parse_command_line()
-    else:
-        try:
-            parse_config_file(options.conf_path, final=False)
-        except IOError:  # configuration file doesn't exist, use defaults
-            pass
+class ContorlledCacheStaticFilesHandler(tornado.web.StaticFileHandler):
```

Comment:
this handler disables caching when working in debug mode. It allows to test js/css changes without app restart